share/governance.md

```diff
@@ -0,0 +1,95 @@
+
+*How to become one*: Be a collaborator and be nominated to the PMC as a contributor. Nominations should be sent to <email address for nominations>. You may nominate yourself. It is also common to give contributor status to an individual who donates code to the project by migrating a repository to the github namespace.
+
+
+Project Management Committee (PMC)
+----------------------------------
+
+
+The project management committee has responsibilities beyond contributors that include participating in strategic planning, release planning, and approving changes to the governance model. They also have the duty of upholding the community code of conduct and ensuring its values. The PMC is also responsible for updating the community code of conduct. It also makes decisions when community consensus cannot be reached. 
+
+The PMC has final say over who can become a committer and will use lazy consensus for approval. Discussion over committer nominations will be done in private.
+
+Membership of the PMC is by election. Condorcet(https://en.wikipedia.org/wiki/Condorcet_method) voting is held once a year. A non-running election official is appointed by the PMC and runs the election. There are 5 members of the PMC. Terms are 1 year.
+
+*How to become one*: Win in the yearly election.
+
+
+
+Lazy Consensus
+--------------
+
+Lazy consensus is a very important concept within the project. It is this process that allows a large group of people to efficiently reach consensus, as someone with no objections to a proposal need not spend time stating their position, and others need not spend time reading such statements.
+
+For lazy consensus to be effective, it is necessary to allow at least 72 hours before assuming that there are no objections to the proposal. This requirement ensures that everyone is given enough time to read, digest and respond to the proposal. This time period is chosen so as to be as inclusive as possible of all participants, regardless of their location and time commitments.
+
+We do require one affirmative vote as part of the Lazy consensus model.
+
+Vox Pupuli has a public mailing list. This list is low-traffic and is for discussion of anything Vox Pupuli related. Controversial patches, nominations for promotion, policy shifts, and anything else can be brought to this list. All contributors and PMC members are expected to be on this list. Release announcements will go out to this list, but will have ``[Release]`` in the subject line so they can be easily filtered.
```

Comment:
It's in here because until now we've gone without a mailing list.(worse, we made one, and no one uses it, and it bears the wrong name)  Adding it here represents a clear break from established procedure and gives weight and authority to the list. 

I do expect we can get along with only one list, but if we need more lists, the PMC can amend the document to enumerate the other lists(or remove references to lists entirely, putting that where it probably truly belongs: in the documentation)